src/processing/spot.rs

```diff
@@ -166,6 +185,15 @@ pub async fn process_data_by_publisher(
     let data: SpotEntry = match config.network().name {
         NetworkName::Testnet => {
             testnet_dsl::spot_entry
+                .filter(testnet_dsl::network.eq("testnet"))
+                .filter(testnet_dsl::publisher.eq(publisher.clone()))
+                .order(testnet_dsl::block_timestamp.desc())
+                .first(&mut conn)
+                .await?
+        }
+        NetworkName::PragmaDevnet => {
+            testnet_dsl::spot_entry
```

Comment:
you should use pragma_devnet_dsl I think

Reply:
yeah sry, done